spec/features/user_edits_a_song_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.feature "User edits a song" do
+  scenario "they see the page for the individual song" do
+    artist = create(:artist)
+
+    old_song_title = "One Love"
+    song_title = "Could You Be Loved"
+
+    visit artist_path(artist)
+    click_on "Edit"
+    fill_in "song_title", with: song_title
```

Comment:
@s-espinosa
@carmer
Any idea why this test is failing? The error reads "Capybara::ElementNotFound; Unable to find field 'song_title'" but it's the same setup and running off of the same partial as user_creates_a_song_spec